gitlab.go

```diff
@@ -497,6 +497,34 @@ func (c *Client) Do(req *http.Request, v interface{}) (*Response, error) {
 	return response, err
 }
 
+// Lint validates .gitlab-ci content
+//
+// GitLab API docs: https://docs.gitlab.com/ee/api/lint.html
+func (c *Client) Lint(content string, options ...OptionFunc) (valid bool, err error) {
```

Comment:
It's convention that only the variables that are part of the API path are directly added in the function signature. Any other params (such as `content` in this case) are part of an options struct.

I know in this case that's maybe a bit much, but I guess I'm just a sucker for consistency :wink: